src/main/java/liuni/resources/LiUniResource.java

```diff
@@ -40,6 +40,9 @@ public Response fetchTimeline() {
                           .map(list -> Response.noContent()
                                              .type(MediaType.APPLICATION_JSON)
                                              .status(Response.Status.OK)
+                                             .header("Access-Control-Allow-Origin", "*")
```

Comment:
lots of copy/paste duplication per call. This should be done in a more central area